style/style_module.css

```
@@ -0,0 +1,331 @@
+}
+.nav-top .nav-top__btn--more {
+    float: right;
+    width: 24px;
+    height: 24px;
+}
+
+.nav-top .nav-top__img-icon-arrow-left,
+.nav-top__img-icon-more {
+    width: 100%;
+    height: 100%;
+}
+
+.nav-top .nav-top__txt-home-feed {
+    font-size: 18px;
+    font-weight: 500;
+}
+
+.nav-top .nav-top__inp-sch {
+    width: 100%;
+    box-sizing: border-box;
+    height: 32px;
+    margin-left: 25px;
+    padding: 7px 16px;
+    background-color: #f2f2f2;
+    border: solid 1px #f2f2f2;
+    border-radius: 32px;
+    font-size: 14px;
+}
+
+.nav-top .nav-top__chat-username {
+    display: inline-block;
+    vertical-align: text-top;
+}
+
+
+/* 하단 tab-menu */
+.nav-tab-menu {
+    border-top: solid 1px #DBDBDB;
+    height: 60px;
+    width: 100%;
+}
+.nav-tab-menu .nav-tab-menu__list {
+    display: flex;
+    justify-content: space-around;
+    height: 100%;
+}
+.nav-tab-menu .nav-tab-menu__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 84px;
+    box-sizing: border-box;
+}
+.nav-tab-menu .nav-tab-menu__wrapper {
+    text-align: center;
+    text-decoration: none;
+    color: #767676;
+    width: inherit;
+}
+.nav-tab-menu__item .nav-tab-menu__wrapper:hover {
+    text-decoration: underline;
+}
+.nav-tab-menu__item .nav-tab-menu__txt {
+    font-size: 11px;
+    line-height: 14px;
+    margin-top: 6px;
+}
+
+.nav-tab-menu__wrapper:before {
+    display: block;
+    content: '';
+    margin: 0 auto;
+    width: 24px;
+    height: 24px;
+}
+
+.nav-tab-menu__wrapper.nav-tab-menu__home:before {
+    background-image: url("../images-module/icon-home.png");
+}
+.nav-tab-menu__wrapper.nav-tab-menu__home.nav-tab-menu__home--active:before {
+    background-image: url("../images-module/icon-home-fill.png");
+}
+
+.nav-tab-menu__wrapper.nav-tab-menu__chat:before {
+    background-image: url("../images-module/icon-message-circle.png");
+}
+.nav-tab-menu__wrapper.nav-tab-menu__chat.nav-tab-menu__chat--active:before {
+    background-image: url("../images-module/icon-message-circle-fill.png");
+}
+
+.nav-tab-menu__wrapper.nav-tab-menu__post:before {
+    background-image: url("../images-module/icon-edit.png");
+}
+
+.nav-tab-menu__wrapper.nav-tab-menu__profile:before {
+    background-image: url("../images-module/icon-user.png");
+}
+.nav-tab-menu__wrapper.nav-tab-menu__profile.nav-tab-menu__profile--active:before {
+    background-image: url("../images-module/icon-user-fill.png");
+}
+
+.nav-tab-menu__txt.nav-tab-menu__txt--active {
+    color: #8D334C;
+}
+
+
+/* 댓글 입력창(comment) */
+.module-inp-cmmt {
+    border: solid 1px #DBDBDB;
+    padding: 0 20px 0 16px;
+    height: 61px;
+    color: #c4c4c4;
+    display: flex;
+    align-items: center;
+}
+
+.module-inp-cmmt img {
+    margin: auto 0;
+}
+
+.module-inp-cmmt__form {
+    width: 100vw;
+}
+
+.module-inp-cmmt .module-inp-cmmt__inp {
+    border: none;
+    margin-left: 18px;
+    height: 18px;
+    width: calc(100% - 100px);
+}
+
```

댓글 focus시 이런 방향으로 가보는건 어떨까요?! 단순한 의견입니다! ㅎㅎㅎ
```suggestion
.module-inp-cmmt .module-inp-cmmt__inp:focus {
  outline: none;
  border-bottom: 1px solid #000;
}
```

---

> 댓글 focus시 이런 방향으로 가보는건 어떨까요?! 단순한 의견입니다! ㅎㅎㅎ

좋네요 UI적으로 더 깔끔할것 같아요 의견 감사합니다 반영하겠습니다!